package.json

```diff
@@ -55,7 +55,7 @@
     "handlebars": "^4.7.4",
     "joi": "^17.0.0",
     "legacy-readable-stream": "npm:readable-stream@^1.0.34",
-    "typescript": "^4.9.3"
+    "typescript": "^4.9.4"
```

Comment:
I'm wondering if we should add that to the matrix, coding for the very latest typescript is not always what people use.